src/igvfd/tests/fixtures/schemas/open_reading_frame.py

```diff
@@ -42,8 +42,19 @@ def orf_zscan10(testapp, gene_zscan10_mm, award, other_lab):
 
 
 @pytest.fixture
-def open_reading_frame_v1(orf_zscan10):
+def open_reading_frame_v2(orf_zscan10):
+    """This is to generate an example at Schema v2"""
     item = orf_zscan10.copy()
+    item['gene'] = item.pop('genes', None)
+    item.update({
+        'schema_version': '2',
+    })
+    return item
+
+
+@pytest.fixture
+def open_reading_frame_v1(open_reading_frame_v2):
+    item = open_reading_frame_v2.copy()
```

Comment:
Later version fixtures should go below earlier version ones. Also earlier version fixtures shouldn't copy later ones.

Reply:
Moved the codes and added a note to the fixture v2 for test. 

Note: The test upgrade uses `f'{notes}. <text text text>.'` format, but this may end up resulting in the existing notes ending in 2 periods. I replicated that in v2->v3 upgrader test, not sure whether that is intentional or not.